backend/lib/services/cloudprofiles.js

```diff
@@ -31,19 +32,22 @@ function fromResource ({ cloudProfile: { metadata, spec }, seeds }) {
   return { metadata, data }
 }
 
-function fromSeedResource ({ metadata, spec }) {
+function fromSeedResource ({ metadata, spec }, gardenShoots) {
   metadata = _.pick(metadata, ['name'])
   const provider = _.get(spec, 'provider')
   const volume = _.get(spec, 'volume')
   const data = { volume, ...provider }
-  return { metadata, data }
+  const canGetShootForSeed = _.some(gardenShoots, shoot => {
+    return shoot.metadata.name === metadata.name
+  })
+  return { metadata, data, canGetShootForSeed }
```

Comment:
Why do we introduce a new property on top level. The idea has been to have metadata and data. Why not have this in metadata `isShootedSeed` or something like this.

Reply:
we can move it to metadata. The reason for the naming is that `isShootedSeed` indicates that this value gives information about this. Actually, as we make the call with the user token, an admin may receive `true` while a regular user would receive `false`, as he cannot access the `garden` namespace. So the only real information is, if we can _access_ the shoot resource in the `garden` namespace.